src/code42cli/cmds/detectionlists/__init__.py

```diff
@@ -13,6 +13,11 @@ def update_user(sdk, username, cloud_alias=None, risk_tag=None, notes=None):
     """
     user_id = get_user_id(sdk, username)
     if cloud_alias:
```

Comment:
The UI only allows you to have one additional alias... This removes all "additional" aliases before adding the new one, to mirror the Code42 UI.